Enrique-e-morales/week6/monument-app/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import './App.css';
+import NavBar from './NavBar/NavBar';
+import Footer from './Footer/Footer';
+import Index from './Index/Index';
+import Blog from './Blog/Blog';
+
+class App extends Component {
+    render(){
+        return(
+            <div className="App">
+                <Index />
```

Comment:
The order of your components is a bit out of whack. I would adjust them to match the flow of your site.